apps/pano/app/root.tsx

```diff
@@ -107,6 +120,28 @@ const Document = () => {
         />
       </head>
       <body className={theme === "DARK" ? darkTheme : ""}>
+        {process.env.NODE_ENV === "development" || !gaTrackingID ? null : (
```

Comment:
`env.NODE_ENV` did not work here?

Reply:
it doesn't work when I use `env.NODE_ENV` :/

<img width="825" alt="image" src="https://user-images.githubusercontent.com/8138047/223582442-20c32117-20d1-4ca5-bdda-74c17ac2103f.png">